docs/rgw_setup.md

```diff
@@ -26,68 +23,43 @@ ceph-rgw01
 ceph-rgw02
 ceph-rgw03
 ```
-**Special Note**: You will need to have access to the keystone host/container via ssh, from the deployment host
-
 
 ### Preset variables
-rpc-ceph has already assigned some ceph-ansible variable in the file **(playbooks/group_vars/rgws.yml)[https://raw.githubusercontent.com/rcbops/rpc-ceph/master/playbooks/group_vars/rgws.yml]**  If special needs are required for a specific customer's requirements these variable can be changed or overridden
+RPC-ceph has already assigned some ceph-ansible variables in the file **(playbooks/group_vars/rgws.yml)[https://raw.githubusercontent.com/rcbops/rpc-ceph/master/playbooks/group_vars/rgws.yml]**
+
+If you need to override ceph.conf variables use the `ceph_conf_overrides_extra` variable hash within your vars file.
+
+### These variables are required to be populated for **both** use cases.
 
-### Make sure you have each of these variables set in your vars file
 * public_network
-* internal_lb_vip_address
-* service_region
+* internal_lb_vip_address or keystone_service_adminurl
+* service_region or radosgw_keystone_service_region
+* radosgw_keystone ***(set to True)**
 * radosgw_keystone_admin_user
 * radosgw_keystone_admin_password
 * radosgw_keystone_admin_tenant
 
-### Run the deployment
-```bash
-ceph-ansible -i <link to your inventory file> playbooks/deploy-ceph.yml -e@<link to your vars file>
-```
+**NB** If your Auth system is preconfigured user, tenant & password settings need to match the Auth system's settings.
```

Comment:
If your Auth system is preconfigured: user, tenant & password settings need to match the Auth system's settings.